src/ThreeDSAuth.res

```diff
@@ -0,0 +1,94 @@
+open Utils
+let getKeyValue = (json, str) => {
```

Comment:
Can we please move this function to Utils

Reply:
good catch